pennylane_lightning/src/simulators/lightning_qubit/gates/cpu_kernels/GateImplementationsAVXCommon.hpp

```diff
@@ -245,8 +252,8 @@ class GateImplementationsAVXCommon
         PL_ASSERT(wires.size() == 1);
 
         const auto rotMat =
-            (inverse) ? Gates::getRot<PrecisionT>(-omega, -theta, -phi)
-                      : Gates::getRot<PrecisionT>(phi, theta, omega);
+            (inverse) ? getRot<std::complex, PrecisionT>(-omega, -theta, -phi)
+                      : getRot<std::complex, PrecisionT>(phi, theta, omega);
```

Comment:
Is the `std::complex` needed here? Just wondering what the difference is from the prior implementation

Reply:
I just kept the definition which is templated over the complex type template so it works with `Kokkos::complex` too.